pydub/audio_segment.py

```diff
@@ -906,9 +928,11 @@ def export(self, out_f=None, format='mp3', codec=None, bitrate=None, parameters=
         conversion_command = [
             self.converter,
             '-y',  # always overwrite existing files
-            "-f", "wav", "-i", data.name,  # input options (filename last)
+            "-f", "wav",  # input options (filename last)
         ]
 
+        conversion_command.extend(item for sublist in [["-i", data.name] for data in data_files] for item in sublist)
```

Comment:
Is this the correct CLI flag to add multiple inputs? Based on my test, only the first file is included in the output file. Multiple files might require the use of `-f concat`.

See: [h.264 - How to concatenate two MP4 files using FFmpeg? - Stack Overflow](https://stackoverflow.com/questions/7333232/how-to-concatenate-two-mp4-files-using-ffmpeg)

Reply:
A revised method for combining all WAV files.

```python
        # build converter command to export using concat demuxer
        list_file = NamedTemporaryFile(mode='w', delete=False)
        for data in data_files:
            list_file.write(f"file '{data.name.replace('\\', '/')}'\n")
        list_file.close()

        conversion_command = [
            self.converter,
            '-y',  # always overwrite existing files
            '-f', 'concat',
            '-safe', '0',
            '-i', list_file.name
        ]
```